tools/build_full_service.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/python3
+# Copyright (c) 2018-2022 The MobileCoin Foundation
+
+import subprocess
+import os
+
+PROJECT_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
+MOBILECOIN_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', 'mobilecoin'))
+CARGO_FLAGS = '--release'
+LEDGER_BASE = os.path.join(PROJECT_DIR, 'target', "sample_data", "ledger")
+
+print('building full service...')
+cmd=' '.join([
+  'RUST_LOG=debug'
+  'MC_SEED=a4aa76e4a5ca70c8447dd544a63f180b5a6fe0aff96495802506354c10f2886e',
```

Comment:
MC_SEED does not affect anything `cargo build` is doing - I suspect it can be removed.